### DISCUSSION TITLE
Add option to map Spectrum to FN keys in order to reenable NOAA alert feature

### DISCUSSION THREAD

**Discussion by salokinrekced** [egzumer/uv-k5-firmware-custom#556] [Ideas]
Reviving a conversation earlier that resulted in this fork: https://github.com/bogdanbrudiu/uv-k5-firmware-custom

This reenables NOAA alert and allows Spectrum to be added to the FN keys. Please remap the Spectrum keys to allow NOAA to be reenabled and use the F+5 command to access as it is on the stock firmware.

**Comment by cgolchert** [comment]
When I look at that fork it is giving me the same options as the main.  I need to enable NOAA in the makefile in that fork as well.  Which sadly fails to build using the Codespace build method just like the main branch.

> /usr/lib/gcc/arm-none-eabi/14.1.0/../../../../arm-none-eabi/bin/ld: firmware section `.text' will not fit in region `FLASH'
/usr/lib/gcc/arm-none-eabi/14.1.0/../../../../arm-none-eabi/bin/ld: region `FLASH' overflowed by 644 bytes
collect2: error: ld returned 1 exit status
make: *** [Makefile:441: firmware] Error 1
